packages/ra-ui-materialui/src/button/BulkDeleteWithConfirmButton.tsx

```diff
@@ -141,13 +145,20 @@ const sanitizeRestProps = ({
     'resource' | 'icon' | 'mutationMode'
 >) => rest;
 
-export interface BulkDeleteWithConfirmButtonProps
-    extends BulkActionProps,
+export interface BulkDeleteWithConfirmButtonProps<
+    RecordType extends RaRecord = any,
+    MutationOptionsError = unknown
+> extends BulkActionProps,
         ButtonProps {
     confirmContent?: React.ReactNode;
     confirmTitle?: string;
     icon?: ReactElement;
     mutationMode: MutationMode;
+    mutationOptions?: UseMutationOptions<
```

Comment:
the type is incorrect. This type doesn't contain `meta`. use `useEditController` as an inspiration instead.

https://github.com/marmelab/react-admin/blob/9b028c1e0c9ace57630530f6bfbfd0392f3031a1/packages/ra-core/src/controller/edit/useEditController.ts#L230-L234